dbt/models/curated/transfers.sql

```diff
@@ -0,0 +1,31 @@
+with transfers_cte as (
+
+    select * from {{ ref('base_transfers') }}
+
+),
+players_cte as (
+
+    select
+        player_id,
+        name as player_name
+    from {{ ref('players') }}
+
+)
+
+select
+    t.player_id,
+    t.transfer_date,
+    t.transfer_season,
+    t.from_club_id,
+    t.to_club_id,
+    t.from_club_name,
+    t.to_club_name,
+    t.transfer_fee,
+    t.market_value_in_eur,
+    p.player_name
+
+from transfers_cte t
```

Comment:
I usually go with avoiding table aliases, even if it's a bit more verbose, since it's also easier to tell in the select from with tables the columns are coming from.

Not a big deal though.